src/Storefront/Framework/Csrf/CsrfPlaceholderHandler.php

```diff
@@ -45,6 +45,10 @@ public function replaceCsrfToken(Response $response, Request $request): Response
 
         $content = $response->getContent();
 
+        if (empty($content)) {
```

Comment:
I would like to see a explict check here like $response instanceof StreamedResponse.
We should not even call `getContent` on a streamed response.

Reply:
Thanks for your review. I updated that in an amended commit.
My original idea was to also not try CSRF replacement on generally empty results, but like this it's more obvious.